src/SFA.DAS.AANHub.Application/Common/GetMemberResult.cs

```diff
@@ -21,19 +25,31 @@ public class GetMemberResult
         if (member == null)
             return null;
 
+        Guid? apprenticeId = null;
+        if (member.Apprentice != null)
+            apprenticeId = member.Apprentice!.ApprenticeId;
```

Comment:
Could have just used null propagators instead of null negators
```suggestion
            apprenticeId = member.Apprentice?.ApprenticeId;
```

Reply:
Implemented